examples/services.ml

```diff
@@ -1,7 +1,7 @@
 open Lwt
-open V1_LWT
+open Mirage_types_lwt
```

Comment:
I'm about to automatically generate a *lot* of PRs just like this one, so if you feel strongly that we should replace the references to V1 and V1_LWT with specific references to the module type from the package where it lives, now's the time to say so.